src/converter/modules/blocks/create-related-resources.js

```diff
@@ -1,18 +1,29 @@
+import yaml from 'js-yaml';
 import { toBlock, replaceElement } from '../utils/dom-utils.js';
 
-export default function createRelatedResources(document) {
+export default function createRelatedResources(document, meta) {
   const ullistElements = Array.from(
     document.querySelectorAll('main > div > ul'),
   );
+  const fullMetadata = yaml.load(meta);
+  if (fullMetadata.type === 'Documentation') {
+    if (ullistElements.length) {
+      ullistElements.forEach((ullistElement) => {
+        const h2 = ullistElement.previousElementSibling;
+        if (
+          h2.tagName === 'H2' &&
+          (h2.id === 'lists-resources' ||
```

Comment:
These dont need to be blocks. It's just a header followed with a list of links, no need to make it a block.